enterprise.constants

```diff
@@ -9,6 +9,10 @@ KIT_ID='adsk'
 # Define your GitHub Enterprise server
 GITHUB_SERVER='git.yourcompany.com'
 
+
+# Add the protocol used to access GitHub server: http or https
+GITHUB_PROTOCOL='https'
```

Comment:
Plus, I would vote against using the term `PROTOCOL` because that could also mean "SSH"...